tests/playbooks/tests_ethtool_features.yml

```diff
@@ -132,28 +136,34 @@
                 TEST: Change feature with both underscores and dashes.
               debug:
                 msg: "##################################################"
-            - network_connections:
-                provider: "{{ network_provider | mandatory }}"
-                connections:
-                  - name: "{{ interface }}"
-                    state: up
-                    type: ethernet
-                    ip:
-                      dhcp4: "no"
-                      auto6: "no"
-                    ethtool:
-                      features:
-                        tx_tcp_segmentation: "no"
-                        tx-tcp-segmentation: "no"
-                __header: "# Ansible managed test header"
-              register: __network_connections_result
+            - name: Import network role
```

Comment:
This changes the semantic - before, the test is calling the `network_connections` module directly - with the change you are calling the role (which calls the module) - does this work exactly the same way?

Reply:
IIRC, calling the module was done as a proof-of-concept to speed up testing since we do not need to run the whole role for every test.